erizo_controller/erizoController/roomController.js

```diff
@@ -11,9 +11,14 @@ exports.RoomController = function (spec) {
         subscribers = {},
         // {id: erizoJS_id}
         publishers = {},
-        // {erizoJS_id: {publishers: [ids], kaCount: count}}
+
+        // {erizoJS_id:
+        //     {publishers: [ids], kaCount: count, agentId: agentId, internalId: internalId}}
         erizos = {},
 
+        maxErizos = spec.maxConnections || global.config.erizoController.maxErizosInRoom,
```

Comment:
Why use `maxConnections`? It's probably clearer if we use a name closer to the one in the config